rt/features/logging/src/main/java/org/apache/cxf/ext/logging/osgi/Activator.java

```diff
@@ -71,6 +74,14 @@ public void updated(Dictionary config) throws ConfigurationException {
             Long inMemThreshold = Long.valueOf(getValue(config, "inMemThresHold", "-1"));
             Boolean logMultipart = Boolean.valueOf(getValue(config, "logMultipart", "true"));
             Boolean logBinary = Boolean.valueOf(getValue(config, "logBinary", "false"));
+            Set<String> sensitiveElementNames = new HashSet<>(
+                    Arrays.asList(
+                            String.valueOf(getValue(config, "sensitiveElementNames", ""))
```

Comment:
Could we please trim the values here to eliminate leading / trailing spaces issue:

```
Arrays
    .stream(String.valueOf(getValue(config, "sensitiveElementNames", "").split(","))
    .map(String::trim)
    .filter(not(Srtring::isEmpty))
    .collect(Collectors.toSet());
``` 

Thank you.